src/main/java/seedu/flashcard/ui/MainWindow.java

```diff
@@ -241,7 +241,11 @@ private CommandResult executeCommand(String commandText) throws CommandException
                 enterStudyMode(new ReviewPanel(logic, this));
             }
 
-            if (commandResult.getViewIndex() != null) {
+            if (commandResult.isShowStats() && commandResult.getViewIndex() != null) {
+                //TODO
```

Comment:
Is this if block left blank intentionally?

Reply:
@sgn00 will implement this in the UI